nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -195,7 +195,30 @@ GRANT ALL PRIVILEGES ON *.* TO 'mysql'@'localhost' WITH GRANT OPTION;
       The NixOS module system now supports freeform modules as a mix between <literal>types.attrsOf</literal> and <literal>types.submodule</literal>. These allow you to explicitly declare a subset of options while still permitting definitions without an associated option. See <xref linkend='sec-freeform-modules'/> for how to use them.
      </para>
    </listitem>
-  </itemizedlist>
+   <listitem>
+     <para>
+       The <link linked="opt-services.mediatomb">mediatomb service</link>
+       declares new and adapt some options so the configuration generation is
```

Comment:
```suggestion
       declares some new and adapts some of the old options so the configuration generation is
```

Grammar seemed off, but I might be wrong (I'm not a native speaker myself).

Reply:
Thanks, it's off alright.
Improving.